packages/bpk-docs/src/layouts/DefaultLayout/DefaultLayout.js

```diff
@@ -61,7 +67,7 @@ class DefaultLayout extends Component {
     const { children } = this.props;
 
     return (
-      <div>
+      <EnhancedThemeProvider themeAttributes={[...themeAttributes]}>
```

Comment:
So here's where it gets interesting: I'm wrapping the whole Layout with the theme provider, the upside is that this will work across the board, the downside is that we have to provide a value for each themable attribute, there are two ways we can go: 
1. we leave like this and we'll provide all the themable attributes
2. we wrap each component with the theme provider

Thoughts?

Reply:
Well yeah, this is what consumer have to do...

We import the `themeAttributes` directly from the component packages that we wanna theme and spread them to theme provider.

Reply:
Since it's shared by the docs and the stories having all themeable attributes everywhere is fine imo

Reply:
I'm leaning towards the first option - as I feel that's the most optimum use of the theme provider, as one big wrapper around everything you want themed. It feels a bit redundant to wrap everything with it.

Reply:
so we all agree on option 1?